aptos-node/src/lib.rs

```diff
@@ -278,6 +278,34 @@ where
         // Build a single validator network with a generated config
         let mut template = NodeConfig::default_for_validator();
 
+        // Adjust some fields in the default template to lower the overhead of running on a
+        // local machine
+        template.state_sync.storage_service.max_concurrent_requests = 2;
+        template
+            .state_sync
+            .aptos_data_client
+            .max_num_in_flight_priority_polls = 2;
+        template
+            .state_sync
+            .aptos_data_client
+            .max_num_in_flight_regular_polls = 2;
+        template
+            .state_sync
+            .aptos_data_client
+            .summary_poll_interval_ms = 1000;
+        template
+            .state_sync
+            .storage_service
+            .storage_summary_refresh_interval_ms = 1000;
+        template
+            .state_sync
+            .state_sync_driver
+            .max_consecutive_stream_notifications = 2;
+        template.execution.concurrency_level = 2;
```

Comment:
it's fine to use 1 too

Reply:
Yeah,  1 will have smaller overhead,  while not being slower when there are only a few transactions

Reply:
I would like to leave it at 2 so as to trigger parallel execution in local mode testing - otherwise, IIUC, the code follows a different path which might lead to different behavior in local mode vs prod.

Reply:
I don't think it's the responsibility of developers using local testnet to test parallel execution vs sequential execution. they should assume it's guaranteed to be the same result

Reply:
You are right but my concern is, given we have enabled parallel execution by default for all testing and prod workload, there is no guarantee that the serial execution path still works as expected (and continues to work going forward). I think the ideal case is we clean up the code for serial execution and use the same code path even when `concurrency_level` is set to 1. Is there any blocker for us to clean up the serial execution or is it just a matter of bandwidth?

Reply:
we want the sequential execution to verify the correctness of the parallel execution I think